### ISSUE TITLE
Can't make it work

### ISSUE THREAD

**Issue by cicerohellmann** [Ramotion/folding-cell#109]
I really would like a better tutorial, I really liked your lib, I really would like to try it out!

**Comment by Jauzee** [comment]
@cicerohellmann did you try a demo app? If you have some issues, please try to start from demo project. We'll try to make docs more easier to follow on.

**Comment by rwever-316** [comment]
I've tried for 2 days to get this to work.  I've tried by:

1. importing project into xcode
2. using CocoaPods with Podfile
3. just dragging and dropping FoldingCell.swift file to my project

Still no luck.  A more complete video and/or better docs would be great.

Thus far the only working copy I have is from Ramotion on the Apple AppStore.

Any one have some words of wisdom? (I'm using xcode 8.1, swift 3)
